### ISSUE TITLE
Feedback: Excalidraw for note-taking

### ISSUE THREAD

**Issue by vguillet** [zsviczian/obsidian-excalidraw-plugin#933]
*Note: This post is meant as constructive feedback/criticism, I am genuinely grateful for the incredible work and effort that has been put into this plugin so far, and it in no way should reflect negatively on the work. This is just feedback from a user to try and provide useful information to help the devs.*

Ever since excalidraw came out, I have been following the project with significant interest. I am an avid obsidian user who came from OneNote, so naturally, the thought of adding freehand writing to obsidian is nothing short of a dream for me.

Despite all of that, I can seem to have been able to properly adopt excalidraw. Upon reflecting on it, I find it just does not correspond to my use case. What makes OneNote so good is the simplicity and fluidity of the writing process (a great example of less is more). When taking notes, all this really mater is having a few pens presets (for consistency), easy color/size adjustments, the ability to draw straight lines (a.k.a. having a ruler), and easy switching of page lines/grid. Everything else comes second in my opinion.

So to summarise, excalidraw is already capable of performing all of the above, but I feel like what is holding it back is its user interface. As of now, excalidraw's interface reminds me more of a photoshop-like approach, where all aspects and tweaks are constantly presented to the user (despite not being necessary for a more note-taking scenario). It as a result seems more tailored for drawing diagrams than note writing.

Solution:
Offer an alternative, more note-focused interface, which similarly to OneNote (apologies for constantly referring to it, but I believe it to still remain a key reference in the field), provides **a notebook and pencil case-like interface**, with an easy definition of pen presets, a digital ruler and page lines/line style toggling.

**Comment by zsviczian** [comment]
Have you tried "tray-mode" (you can access it via the command palette)?
Have you tried zen-mode (you can access it via the context menu)?

However - Excalidraw is designed as a quick vector graphics illustration app with a hand-drawn feel, not as a handwritten (i.e. with a stylus) note-taking app. While some handwritten text is possible, Excalidraw is inefficient with long stretches of handwritten notes (to the extent that it freezes on an iOS device due to some Safari compatibility issues). Also, the pen is not very configurable (technically perfect-freehand, the package used by Excalidraw) is more configurable, however, the developers of the core excalidraw package have decided to hide/limit those features in Excalidraw.

I also use onenote in the corporate setting, I am familiar with its strengths and weaknesses when taking handwritten notes. It is a reasonable tool for handwritten notes, though I used much better apps - focusing now on handwriting specifically.

**Comment by vguillet** [comment]
Thank you for the reply, I will look into all your suggestions! If you also had suggestions on better alternatives to OneNote I would also be interested! Thank you for your contribution to the community!

**Comment by zsviczian** [comment]
Consolidating pen / stylus / handwriting / drawing tablet support discussion into #2773.

Closing this as a duplicate so related requests stay centralized. Please continue discussion there.
